packages/react-dom/src/__tests__/ReactDOMEventListener-test.js

```diff
@@ -211,25 +236,30 @@ describe('ReactDOMEventListener', () => {
     }
 
     const container = document.createElement('div');
-    const instance = ReactDOM.render(<Wrapper />, container);
+    const root = ReactDOMClient.createRoot(container);
+    await act(() => {
+      root.render(<Wrapper />);
+    });
+
+    const instance = container.firstChild.firstChild;
```

Comment:
It seems to me `getInner()` returned the inner div. `container.firstChild.firstChild` should be `<div id="outer" />`
```jsx
// container
<div> // container.firstChild
  <div onMouseOut={onMouseOut} id="outer"> // container.firstChild.firstChild
    <div ref={this.innerRef}> // container.firstChild.firstChild.firstChild
     Inner
    </div>
  </div>
</div>
```

Maybe go with a more verbose ref pattern and get the instance of `Wrapper` and continue using `instance.getInner()`?

Or add `id="inner"`, remove `getInner` and use `document.getElementByID` like we do for `id="outer"`

Reply:
Oh, nice catch, I just moved the createRef outside of the component like we do in other tests.